plutus-cbor/test/Plutus/Codec/CBOR/EncodingSpec.hs

```diff
@@ -401,7 +365,7 @@ genValue = do
 
 genAdaOnlyValue :: Gen Plutus.Value
 genAdaOnlyValue = do
-  n <- genInteger `suchThat` (> 0)
+  n <- genInteger `suchThat` (\x -> x > 0 && x < 45_000_000_000)
```

Comment:
Should: Why 45 Bn Lovelaces limit? This looks like a magic constant that would benefit from being named.

Reply:
I changed it, but for the sake of explaining: 45B is the max ADA supply, so that's basically the biggest tx out quantity we can ever encounter. Though this was even wrong because those are lovelace, to the correct value is `45_000_000_000_000_000` !